README.md

```diff
@@ -44,7 +44,9 @@ Note, you can use any of the supported [storage backends](https://restic.readthe
    * `/etc/restic/default.env` - Profile specific environment variables (multiple profiles can be defined by copying to `/etc/restic/something.env`).
    * `/etc/restic/backup_exclude` - List of file patterns to ignore. This will trim down your backup size and the speed of the backup a lot when done properly!
 1. Initialize remote repo as described [below](#3-initialize-remote-repo)
-1. Enable automated backup for starting with the system:
+1. Configure [how often]https://www.freedesktop.org/software/systemd/man/systemd.time.html#Calendar%20Events() back up should be made.
```

Comment:
Shouldn't `[how often]https://...()` become `[how often](https://...)`? ;)

Reply:
Yesyes! I've not learned to work properly with the new auto-close plugin in my editor that I use yet. thx!